assets/js/modules/analytics/components/common/AccountCreate/index.js

```diff
@@ -58,16 +60,17 @@ export default function AccountCreate() {
 	const canSubmitAccountCreate = useSelect( ( select ) => select( STORE_NAME ).canSubmitAccountCreate() );
 	const isDoingCreateAccount = useSelect( ( select ) => select( STORE_NAME ).isDoingCreateAccount() );
 	const hasProvisioningScope = useSelect( ( select ) => select( CORE_USER ).hasScope( PROVISIONING_SCOPE ) );
+	const hasEditScope = useSelect( ( select ) => isGA4enabled ? select( CORE_USER ).hasScope( EDIT_SCOPE ) : undefined );
```

Comment:
No need to select this conditionally since we're already using the same selector above for the provisioning scope so the resolver will be invoked for `getAuthentication()` anyways. Better to keep it simple and consistent.

Also, this would also cause the `! hasEditScope` to always be `true` below if GA4 was not enabled which could cause problems later if `isGA4enabled` wasn't checked before using it.

```suggestion
	const hasEditScope = useSelect( ( select ) => select( CORE_USER ).hasScope( EDIT_SCOPE ) );
```